appyters/Tumor_Gene_Target_Screener/Tumor_Gene_Target_Screener.ipynb

```diff
@@ -589,16 +597,101 @@
     "        ),\n",
     "        mean=df_bg_stats.loc[(c, 'mean')],\n",
     "        sd=df_bg_stats.loc[(c, 'std')],\n",
-    "        x=df_bg_stats.columns,\n",
+    "        y=df_bg_stats.columns,\n",
     "        name='Background',\n",
+    "        orientation='h'\n",
     "    ))\n",
     "    fig.add_trace(go.Box(\n",
-    "        y=df_expr_norm.loc[c],\n",
+    "        x=df_expr_norm.loc[c],\n",
     "        name='Patient (Normalized)',\n",
+    "        orientation='h'\n",
     "    ))\n",
-    "    fig.update_layout(title=c, autosize=True)\n",
+    "    fig.update_layout(title=c, height=1200)\n",
     "    fig.show()"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c2fc4d4f",
+   "metadata": {},
+   "source": [
+    "## Review Protein Expression Levels of Selected Candidates\n",
```

Comment:
To prevent this section from appearing when the option is toggled off, you should make this a code cell with the source:

```python
%%appyter markdown
{% if proteomics_vis.raw_value %}
## Review Protein Expression ...
...
{% endif %}
```